packages/playwright-crawler/src/internals/utils/playwright-utils.ts

```diff
@@ -25,6 +25,7 @@ import { LruCache } from '@apify/datastructures';
 import log_ from '@apify/log';
 import type { Request } from '@crawlee/browser';
 import { validators, KeyValueStore, RequestState } from '@crawlee/browser';
+import { Configuration } from '@crawlee/core';
```

Comment:
this is an undeclared dependency, import it from the `@crawlee/browser` package instead